crates/examples/push-cdn/marshal.rs

```diff
@@ -0,0 +1,53 @@
+//! The following is the main `Marshal` binary, which just instantiates and runs
+//! a `Marshal` object with the `HotShot` types.
+//!
+use anyhow::{Context, Result};
+use cdn_broker::reexports::connection::protocols::Quic;
+use cdn_marshal::{ConfigBuilder, Marshal};
+use clap::Parser;
+use hotshot::traits::implementations::WrappedSignatureKey;
+use hotshot_example_types::node_types::TestTypes;
+use hotshot_types::traits::node_implementation::NodeType;
+
+//TODO: for both client and marshal, clean up and comment `main.rs`
```

Comment:
Does it make sense for these to be cleanup tasks?

Reply:
Totally forgot to remove this, thanks. I did the cleanup already